x/gamm/pool-models/balancer/amm_test.go

```diff
@@ -833,6 +833,96 @@ func TestCalcJoinPoolShares(t *testing.T) {
 			),
 			expectShares: sdk.NewInt(1250000000000 + 609374990000),
 		},
+		{
+			name:    "minimum input with two assets and minimum liquidity",
```

Comment:
Is there a Wolfram link I could use to verify this?

Reply:
This one doubles the liquidity in the pool from (1uosmo, 1uatom) to (2uosmo, 2uatom), so it will generate the base number of LP shares for a pool upon creation which is 1e20 (or 100 * 10^18).

I'll go ahead and add a comment to clarify but we can reach the correct final result just by using this property of Balancer pools